packages/console/src/components/Executions/Tables/NodeExecutionRow.tsx

```diff
@@ -66,18 +67,20 @@ export const NodeExecutionRow: React.FC<NodeExecutionRowProps> = ({
     ? isEqual(selectedExecution, nodeExecution)
     : false;
 
-  const expanderContent = (
-    <div className={styles.namesContainerExpander}>
-      {isParentNode(nodeExecution) ? (
-        <RowExpander
-          expanded={expanded}
-          onClick={() => onToggle(node.id, node.scopedId, nodeLevel)}
-        />
-      ) : (
-        <div className={styles.leaf} />
-      )}
-    </div>
-  );
+  const expanderContent = React.useMemo(() => {
+    return isParentNode(nodeExecution) ? (
+      <RowExpander
+        ref={ref as React.ForwardedRef<HTMLButtonElement>}
+        key={node.scopedId}
+        expanded={isExpanded(node)}
+        onClick={() => {
+          onToggle(node.id, node.scopedId, nodeLevel);
+        }}
+      />
+    ) : (
+      <div className={styles.leaf} />
+    );
+  }, [node, nodeLevel]);
```

Comment:
@ursucarina we need to duplicate this in timeline tab (`TaskNames.tsx`), or maybe move it to a tiny component which we can use here and in TaskNames